### PR TITLE
Modernized accessor syntax in several places

### PR DESCRIPTION
**Pull request by Lehonti**
## What does the pull request do?
Modernizes syntax of single-expression `get`ters and `set`ters in several files of:
- `Avalonia.Markup.UnitTests`
- `Avalonia.Markup.Xaml.UnitTests`
- `Avalonia.ReactiveUI.UnitTests`
- `Avalonia.RenderTests`

## What is the updated/expected behavior with this PR?
No behavior change

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0040042-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

File: tests/Avalonia.RenderTests/Media/ImageBrushTests.cs [hunk 1]
```diff
@@ -19,15 +19,9 @@ public ImageBrushTests()
         {
         }
 
-        private string BitmapPath
-        {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
-        }
+        private string BitmapPath => System.IO.Path.Combine(OutputPath, "github_icon.png");
```
Comment (robloo) [inline tests/Avalonia.RenderTests/Media/ImageBrushTests.cs:22]: IMO this is a step too far. It's less easy to read. The below would have been enough. Yes, this is dev preference which is why I would change less actually. Other people will have different opinions I'm sure.

```
        private string BitmapPath
        {
            get => System.IO.Path.Combine(OutputPath, "github_icon.png");
        }
```

File: tests/Avalonia.RenderTests/Media/VisualBrushTests.cs [hunk 2]
```diff
@@ -19,18 +19,11 @@ public VisualBrushTests()
...
         }
 
-        private string BitmapPath
-        {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
-        }
+        private string BitmapPath => System.IO.Path.Combine(OutputPath, "github_icon.png");
 
-        private Control Visual
+        private Control Visual => new Panel
```
Comment (robloo) [inline tests/Avalonia.RenderTests/Media/VisualBrushTests.cs:24]: If there is a code block in a getter you really shouldn't be "modernizing" it IMO. It is entirely developer preference in that case.

**Comment by Lehonti** [comment]
Hi @robloo, I figured that since the files will be changed, I could make small changes (like getter modernization) on the way; but they are not strictly necessary. The focus of the pull request is the sealing of the classes, so I could do without the getter modernization.

**Comment by Lehonti** [comment]
Hi @robloo I just toned down the changes a little. Specifically, the getters that perform calculations are back to a block body

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0040050-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

File: tests/Avalonia.RenderTests/Media/ImageBrushTests.cs [hunk 3]
```diff
@@ -21,12 +21,18 @@ public ImageBrushTests()
 
         private string BitmapPath
         {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
+            get
+            {
+                return System.IO.Path.Combine(OutputPath, "github_icon.png");
```
Comment (robloo) [inline tests/Avalonia.RenderTests/Media/ImageBrushTests.cs:26]: To be clear these one-liners make sense to use `get => System.IO.Path.Combine(OutputPath, "github_icon.png");`

I was referring to the cases where there are multiple lines of code and the cases where you removed the `get` accessor entirely.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0040484-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**